assets/_scss/components/_footer.scss

```diff
@@ -161,11 +173,23 @@ li.usa-footer-primary-content {
 }
 
 .usa-footer-secondary_section {
-  background-color: #e0e0e0;
+  background-color: $color-primary-darkest;
+  color: $color-white;
   padding: {
     top: 3rem;
     bottom: 3rem;
   }
+
+  a {
+    color: $color-white;
+
+    &:hover {
```

Comment:
Sorry this is so late @kaelig -  :focus state for links is defined in `src/elements/_typography.scss`.